src/DataTest.java

```diff
@@ -0,0 +1,37 @@
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+/**
+ * Junit testing for the all the data to be read in the system.
+ * @author Azizul Hasan 101124159
```

Comment:
I know you did it but we probably shouldn't be taking credit for specific parts of the program lol. Put our group number or all our names instead

Reply:
I changed it to group